ozkour-back/google-api/slideDataOrganizer.js

```diff
@@ -129,6 +129,48 @@ function tryToFillWithEvent (yNextElmt, allEventsInADate, date, data) {
   return { data: dataOrganizedBySlides, yNextElmt, atLeastOneEventCanFit }
 }
 
+function tryToFillWithTalk (yNextElmt, allTalksInAnEvent, date, event, data) {
+  const dataOrganizedBySlides = data.slice(-1)
+  let i = 0
+  let listOfTalksThatFits = []
+  let atLeastOneTalkCanFit = false
+  let worthToContinue = (i === 0 || atLeastOneTalkCanFit)
+  let DoesTalkFits = true
+
+  while (i < allTalksInAnEvent.length && worthToContinue) {
+    worthToContinue = (i === 0 || atLeastOneTalkCanFit)
```

Comment:
On n'a à priori pas besoin du 'worthToContinue'
```suggestion
  let atLeastOneTalkCanFit = true
  let DoesTalkFits = true

  while (i < allTalksInAnEvent.length && atLeastOneTalkCanFit) {
```

Reply:
J'avais fait ça pour rendre le code plus facile à comprendre